test/helpers/kubectl.go

```diff
@@ -593,6 +594,29 @@ func (kub *Kubectl) labelNodes() error {
 	return nil
 }
 
+// GetCiliumEndpoint returns the CiliumEndpoint for the specified pod.
```

Comment:
Doesn't this duplicate `CepGet` functionality?

Reply:
Yes, pretty much. I've folded the two together.